### PR TITLE
Task/3w4k4g8

### PR DESCRIPTION
**Pull request by awais-supernovabrands**
## Description

Added Table of Contents and ‘back to top’ Button to Blog Articles

### CODE REVIEW

File: src/scss/templates/_blog.scss [hunk 1]
```diff
@@ -112,6 +112,24 @@
 			background-color: white !important;
 		}
 	}
+
+	.blog-post-table {
```
Comment (foria) [inline src/scss/templates/_blog.scss:116]: too much nesting please max 3, but I think you can do that with 2 too

File: src/scss/templates/_blog.scss [hunk 2]
```diff
@@ -160,3 +178,32 @@
...
+	opacity: 0;
+	transition: 0.4s ease-in;
+	transition-property: opacity, bottom;
+	display: none;
+
+	&:hover {
+		color: $primary;
+	}
+
+	body.btt-btn--show & {
```
Comment (foria) [inline src/scss/templates/_blog.scss:197]: how are you going to add that class to the body?

**Review by foria — CHANGES_REQUESTED** [review]

File: src/scss/templates/_blog.scss (hunk 2 above)
Comment (awais-supernovabrands) [reply src/scss/templates/_blog.scss:197]: > how are you going to add that class to the body?

I just reused the existing scrolling function. On scroll, this class is added.

File: src/scss/templates/_blog.scss (hunk 1 above)
Comment (awais-supernovabrands) [reply src/scss/templates/_blog.scss:116]: > too much nesting please max 3, but I think you can do that with 2 too

Ok, let me modify it again.

**Comment by awais-supernovabrands** [comment]
The requested changes have been modified and completed.

File: src/scss/templates/_blog.scss [hunk 3]
```diff
@@ -160,3 +160,46 @@
...
+	opacity: 0;
+	transition: 0.4s ease-in;
+	transition-property: opacity, bottom;
+	display: none;
+
+	&:hover {
+		color: $primary;
+	}
+
+	&.btt-btn--show {
```
Comment (foria) [inline src/scss/templates/_blog.scss:193]: class can be simplie btn--show

File: src/scss/templates/_blog.scss [hunk 4]
```diff
@@ -160,3 +160,46 @@
...
+		}
+	}
+}
+
+.blog-back-to-top {
+	position: fixed;
+	bottom: calc-em(20);
+    right: calc-em(15);
+    cursor: pointer;
+    z-index: $zindex-dropdown;
```
Comment (foria) [inline src/scss/templates/_blog.scss:183]: please check indetation

File: src/scss/templates/_blog.scss [hunk 5]
```diff
@@ -160,3 +160,46 @@
...
+		display: flex;
+	}
+
+	.svg {
+		font-size: calc-em(40);
+	}
+
+	@include media-breakpoint-up(lg) {
+		right: calc-em(40);
+	}
```
Comment (foria) [inline src/scss/templates/_blog.scss:204]: this should go after the main class properties not at its end

**Review by foria — CHANGES_REQUESTED** [review]

File: src/scss/templates/_blog.scss (hunk 3 above)
Comment (awais-supernovabrands) [reply src/scss/templates/_blog.scss]: > class can be simplie btn--show

Class is renamed

File: src/scss/templates/_blog.scss (hunk 5 above)
Comment (awais-supernovabrands) [reply src/scss/templates/_blog.scss]: > this should go after the main class properties not at its end

Breakpoint has been moved

**Pull request merged**